pkgs/tools/misc/venus/default.nix

```diff
@@ -5,11 +5,12 @@ let
 in
 
 stdenv.mkDerivation rec {
-  name = "venus-${rev}";
+  pname = "venus";
+  version = "unstable-2011-02-18";
 
   src = fetchurl {
```

Comment:
We can just use `fetchFromGitHub` which uses the tarball internally. Also https://github.com/rubys/venus/tarball/9de21094a8cf565bdfcf75688e121a5ad1f5397b is a tar.gz. GitHub does not support bzip tarballs afaik.

Reply:
I just changed the version. It does build, maybe it's just by accident that the file is called .tar.bz and still extracted because `tar -xf` is used. I don't know, but can we fix that in another PR?

Reply:
OK, done.